### ISSUE TITLE
Default nodes

### ISSUE THREAD

**Issue by Fantu** [BetaCoinDev/betacoin#6]
I did a quick search of hardcoded default nodes to set it and have betacoin working without mandatory betacoin.conf but I not found where are setted.
Would be good also to add at least one node ipv6 or at least write it in the documentation to allow the use of betacoin also to IPv6-only (although at the moment there are very few).

**Comment by BetaCoinDev** [comment]
thank you for your support!, all issues will be improved in next update

**Comment by Fantu** [comment]
Probably I found where are setted.
Seems there are already some dns setted here:
https://github.com/BetaCoinDev/betacoin/blob/master/src/chainparams.cpp
vSeeds.push_back(CDNSSeedData("seed1.betacoin.org", "seed1.betacoin.org"));
...
but seed*.betacoin.org probably not link to betacoin nodes running.

Probably the problem will easy solved creating/changing the record A "seed1"of betacoin.org domain to ip 66.172.10.57 and "seed2" to ip 76.74.177.244 and record AAAA to ipv6 node.

**Comment by BetaCoinDev** [comment]
yes, seed*.betacoin.org was included for future nodes ipv4 & ipv6 both.
Currently seed1.betacoin.org provides first connection to ipv4 address, i do know that i need to provide more both ipv4 and ipv6 node addresses, but i do not think that is even a minor issue - i mean low number of primary nodes. But yes, definitely more of them will be included in dns records.

**Comment by Fantu** [comment]
the question is, the records are now working or not?
if yes the manual betacoin.conf and "-connect=" on first connect is not needed

**Comment by BetaCoinDev** [comment]
Yes, records are working, you do not need "-connect=" or "-addnode=" on first connection, client will do it automatic. I think betacoin.conf should exist at least for setting up @username and @password, and to customize some options like "server=1 or 0", etc

**Comment by Fantu** [comment]
Good, now probably is good update also the posts you did in some forum about note for first connection

**Comment by BetaCoinDev** [comment]
It was announced in forum https://cryptocointalk.com/topic/2117-first-connection-issue-solved/

**Comment by Fantu** [comment]
There is also first post here to update:
https://bitcointalk.org/index.php?topic=321266.0
and here:
http://betacoin.org/downloads
